src/components/ContentOrSkeleton.tsx

```diff
@@ -0,0 +1,14 @@
+import Skeleton from "@saleor/components/Skeleton";
+import React from "react";
+
+interface ContentOrSkeletonProps {
+  condition: any;
+  children: React.ReactNode;
+}
+
+const ContentOrSkeleton: React.FC<ContentOrSkeletonProps> = ({
+  condition,
+  children
+}) => (!!condition ? <>{children}</> : <Skeleton />);
```

Comment:
I don't like this component. It's good to hide conditional displaying of skeleton, but I would rather think that clearly the main purpose of this component should be to display content, while showing skeleton should be temporary state. Currently the name `ContentOrSkeleton` doesn't suggest it, I would rather see e.g. `WithSkeleton`. Also `condition` prop is not precise, as we don't know clearly what occurs when it is true or false, without seeing implementation details. An interface should clearly define what it expects, e.g. `showSkeleton`.

But going even further, what do you think about even refactoring current `Skeleton` component in `@saleor/components/Skeleton` to be able to accept `show` prop and children, and then use it like:
```tsx
<Skeleton show={!loading}>
  <SomeChildren />
</Skeleton>
```
or use it like it is now when `children` are absent and `show` defaults to true:
```tsx
<Skeleton />
```
?

Reply:
Hiding conditional displaying of the skeleton is a very good idea 👍 .

Reply:
Sounds good to me, @jwm0 does adding this ability to exsiting skeleton sounds ok to you?